app/views/contacts/_form.html.slim

```diff
@@ -0,0 +1,15 @@
+= form_for @contact do |f|
+
+  .form-group
+      = f.text_field :name
+      = f.text_field :address_line_1
+      = f.text_field :address_line_2
+      = f.text_field :town
+      = f.text_field :county
+      = f.text_field :postcode
+      = f.radio_button_fieldset :contact_type, choices: Contact::contact_types.keys, inline: true
```

Comment:
Is there something different about this as normally the radios come out in separate lines?

Reply:
Hmm 🤔 - yeah they do don't they? I'm not sure tbh I just copied from another form. I'll have a look and check if there I can get the default behaviour instead.

Reply:
@digitalali-moj it was the `inline: true` option - not sure why I left this on! removed now and it's in the usual style.